index.html

```diff
@@ -0,0 +1,188 @@
+  </header>
+
+  <main>
+    <!-- Headline Section -->
+
+    <section id="header_section">
+      <div class="wesing_header">
+        <div class="intro">
+          <h2 class="hello">"Hello! Worshipper"</h2>
+          <h2 class="hello_capital">WeSingAfrica Gospel Music Concert 2021</h2>
+        </div>
+
+        <div class="intro_parag">
+          A physical gathering of people from all African countries and a global online reach, with the sole aim of
+          <span>worship</span> through songs is taking place in Lagos, Nigeria, in December.
+        </div>
+
+        <div class="wesing_date">
+          <h2><i class="far fa-calendar-alt"></i> 2021.12.10(FRI) ~ 11(SAT)</h2>
+          <p><i class="far fa-clock"></i> 19:00 ~ 06:00 (UTC+1)</p>
+          <h3><i class="fas fa-map-marker-alt"></i> Teslim Balogun Stadium, Surulere, Lagos.</h3>
+        </div>
+      </div>
+    </section>
+
+    <!-- Main Schedule Section -->
+
+    <section id="project_schedule">
+      <div class="main-program">
+        <h3>Main Program</h3>
+        <hr>
+      </div>
+
+      <div class="wesing_programs">
+        <div class="program">
+          <i class="fas fa-users fa-2x"></i>
+          <h4>Counselling</h4>
+          <p>Get answers to various issues of life from our experienced and spiritually inclined counsellors during our
+            two-hour counselling session</p>
+        </div>
+        <div class="program program_2">
+          <i class="fas fa-pray fa-2x"></i>
+          <h4>Intercession</h4>
+          <p>One hour intercessory prayer for the nations of Africa, governments, global economies, families and
+            individuals. Write your prayer requests.</p>
+        </div>
+        <div class="program program_2">
+          <i class="fas fa-praying-hands fa-2x"></i>
+          <h4>WARship</h4>
+          <p>The Father seeks those who will worship him in spirit and in truth. Get ready to pour your heart out during
+            our four-hour WARship session.</p>
+        </div>
+        <div class="program">
+          <i class="fas fa-volume-up fa-2x"></i>
+          <h4>PraiseParty</h4>
+          <p>Are you ready to danceand sing praises to the Most High? Learn all the amazing dance moves to think you can
+            pull off because we'll be dancing for four hours.</p>
+        </div>
+
+      </div>
+
+      <div class="program_button">
+        <a href="#">See the whole program</a>
+        <button type="submit">Book A Counselling Session</button>
+      </div>
+
+    </section>
+
+    <!-- Invited Guests Section -->
+    <section id="wesing_guests">
+      <div class="main-program">
+        <h3>Guests</h3>
+        <hr>
+      </div>
+
+      <!--Content Of This Section Will Be Displayed Dynamically On Page Load using JavaScript. See guests.js for code.  -->
+      <div class="guest_list"></div>
+
+      <div class="guest_button" id="show_guests">
+        <button type="submit" >More <i class="fas fa-chevron-down"></i></button>
```

Comment:
- [ ] On deployment for mobile size, I'm not able to see the content where I click the `more button`. I invite you to fix it. 🐶 
![image](https://user-images.githubusercontent.com/20563252/128188653-3ede3c74-7298-49fa-88bf-adca1a759b9d.png)